sdk/modelsrepository/azure-iot-modelsrepository/samples/README.md

```diff
@@ -2,7 +2,7 @@
 
 This directory contains samples showing how to use the features of the Azure IoT Models Repository Library.
 
-The pre-configured endpoints and DTMIs within the samples refer to example models that can be found on [devicemodels.azure.com](https://devicemodels.azure.com/). These values can be replaced to reflect the locations of your own models, wherever they may be.
+The pre-configured endpoints and DTMIs within the samples refer to example models that can be found on [devicemodels.azure.com](https://devicemodels.azure.com/) or in the provided local repository [SampleModelRepo](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/modelsrepository/azure-iot-modelsrepository/samples/SampleModelRepo/). These values can be replaced to reflect the locations of your own models, wherever they may be.
```

Comment:
This link looks off, the sample models repo directory is called `SamplesModelsRepo` not `SampleModelRepo`.

Reply:
This link won't exist till the pr is merged in

Reply:
The pipeline appears to resolve the link with respect to the latest PR commit